### DISCUSSION TITLE
Text Formatting not working

### DISCUSSION THREAD

**Discussion by sdowd20** [quarto-dev/quarto-cli#3219] [Q&A] [answered]
HI all! My name is Sally, and I am new to Quarto. I have gone through the Intro to Quarto and authoring workshop. Right now, I'm having pretty basic issues with text formatting that I'm hoping to get resolved.

First off, when I type code and then go to view the Visual tab and return to the Source, some formatting is removed. Namely spaces are removed and a fenced div will go from ::: {.panel-tabset} to ::: panel-tabset. I am wondering why this is. 

When I type to create different size headers with #, it is not working. I'm trying to do #Header 1 ##Header 2 ##Header 3, etc. When I type this into a Quarto doc (outside a code chunk), this format isn't working.

3rd issue: When I try to use a callout-note with this code, it doesn't work:
::: {.callout-note}
Note that there are five types of callouts, including: `note`, `tip`, `warning`, `caution`, and `important`.
:::

I'm having other issues too such as the panel tabset not popping up even when I follow code from the tutorial.

**Comment by mcanouil** [comment, marked as answer]
`::: {.panel-tabset}` is 100 % equivalent to `::: panel-tabset`. Visual editor has to choose one syntax, it can't use both for obvious reason. The brackets syntax is useful when you want to add extra information to the fenced div (styling, attributes).

Quarto is using Markdown which has specific rules, e.g., a space is required between `#` and the title to work, see <https://www.markdownguide.org/cheat-sheet/>.

Regarding callout, please provide a full reproducible example.
To note, using Quarto 1.2.262, the following works just fine:

````md
---
title: Title
format: html
---

```{r}
1 + 1
```

::: {.callout-note}
Note that there are five types of callouts, including: note, tip, warning, caution, and important.
:::
````

**Reply by sdowd20** [reply]
Thank you so much for your help! This information helps a lot to start to be familiar with Quarto. Once I updated my Quarto and followed your reproducible example, the callout portion worked as well.

**Reply by mcanouil** [reply]
I'm glad it helped. 😉
